ts/features/itwallet/presentation/remote/hooks/useItwRemoteUntrustedRPBottomSheet.tsx

```diff
@@ -0,0 +1,62 @@
+import { Body, ListItemHeader, VSpacer } from "@pagopa/io-app-design-system";
+import { View } from "react-native";
+import { memo } from "react";
+import I18n from "../../../../../i18n.ts";
+import { useIOBottomSheetModal } from "../../../../../utils/hooks/bottomSheet.tsx";
+
+/**
+ * Content component for the Not Trusted RP bottom sheet
+ */
+export const ItwRemoteUntrustedRPBottomSheetContent = () => (
```

Comment:
We should remove `export`, since this component is not used anywhere else. Also, wrapping directly the component with `memo` avoids creating two objects.
```suggestion
const ItwRemoteUntrustedRPBottomSheetContent = memo(() => (
```